txpipe/twopoint_fourier.py

```diff
@@ -844,23 +863,45 @@ def save_power_spectra(self, tracers, nbin_source, nbin_lens):
                     i=d.i,
                     j=d.j,
                     n_ell=d.noise[i],
-                    bandpower_index=i,
+                    window_ind=i,
                 )
 
             # Add n_ell_coupled to tracer metadata. This will work as far as
             # the coupled noise is constant
             tr = S.tracers[tracer1]
             if (tracer1 == tracer2) and ("n_ell_coupled" not in tr.metadata):
-                if self.config["analytic_noise"] is False:
-                    # If computed through simulations, it might be better to
-                    # take the mean since, for now, only a float can be passed
-                    i = 0 if "lens" in tracer1 else 2
-                    tr.metadata["n_ell_coupled"] = np.mean(d.noise_coupled)
+
+                if self.config["gaussian_sims_factor"] != [1.] and "lens" in tracer1:
+                    
+                    print (tracer1)
+                    print("ATTENTION: We are multiplying the coupled noise saved in the sacc file by the gaussian sims factor.")
+                    print("Original noise:", d.noise_coupled)
+
+                    if self.config["analytic_noise"] is False:
+                        # If computed through simulations, it might be better to
+                        # take the mean since, for now, only a float can be passed
+                        i = 0 if "lens" in tracer1 else 2
+                        tr.metadata["n_ell_coupled"] = np.mean(d.noise_coupled*self.config["gaussian_sims_factor"][int(tracer1[-1])]**2)
+                    else:
+                        # Save the last element because the first one is zero for
+                        # shear
+                        tr.metadata["n_ell_coupled"] = d.noise_coupled[-1]*self.config["gaussian_sims_factor"][int(tracer1[-1])]**2
+                    print("Noise after scaling:", tr.metadata["n_ell_coupled"])
+                        
                 else:
-                    # Save the last element because the first one is zero for
-                    # shear
-                    tr.metadata["n_ell_coupled"] = d.noise_coupled[-1]
+                    if self.config["analytic_noise"] is False:
+                        # If computed through simulations, it might be better to
+                        # take the mean since, for now, only a float can be passed
+                        i = 0 if "lens" in tracer1 else 2
+                        tr.metadata["n_ell_coupled"] = np.mean(d.noise_coupled)
+                    else:
+                        # Save the last element because the first one is zero for
+                        # shear
+                        tr.metadata["n_ell_coupled"] = d.noise_coupled[-1]
+
+
```

Comment:
To avoid having two ways of writing `tr.metadata["n_ell_coupled"]`, which potentially could lead to bugs, I would leave that part as it was before and modify the `n_ell_coupled` in this line. 

In other words, I would add in this line
```
if self.config["gaussian_sims_factor"] != [1.] and "lens" in tracer1:  
    print (tracer1)
    print("ATTENTION: We are multiplying the coupled noise saved in the sacc file by the gaussian sims factor.")
    print("Original noise:", d.noise_coupled)
    tr.metadata["n_ell_coupled"] *= self.config["gaussian_sims_factor"][int(tracer1[-1])]**2
```